src/components/Reviews.jsx

```diff
@@ -1,35 +1,51 @@
 import React, { useEffect, useState } from "react";
-import { Link } from "react-router-dom";
 import { fetchReviews } from "../Api";
 
 function Reviews() {
   const [currentReviews, setCurrentReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
     fetchReviews().then(({ data }) => {
-      console.log(data.reviews);
       setCurrentReviews(data.reviews);
+      setIsLoading(false);
     });
   }, []);
 
+  if (isLoading) {
+    return <p>Loading...</p>;
+  }
+
   return (
-    <main>
-      <ul className="reviewCard">
-        {currentReviews.map((review) => (
-          <li key={review.id}>
-            <img className="reviewImage" src={review.review_img_url} alt={review.title} />
-            <ul>
-              <li>TITLE: {review.title}</li>
-              <li>DESIGNER: {review.designer}</li>
-              <li>OWNER: {review.owner}</li>
-              <li>VOTES: {review.votes}</li>
-              <li>COMMENTS: {review.comment_count}</li>
-            </ul>
-          </li>
-        ))}
-      </ul>
+    <main className="reviews-container">
```

Comment:
Going forward I would look to refactor this into another component (such as article card) as it will make your code a bit more dry :)